### PR TITLE
Add nss-mdns requested by redis team

### PR DESCRIPTION
**Pull request by 0xba1a**
- Add new package nss-mdns to Mariner
 - Bring authselect to core as it is direct dependency for nss-mdns
 - Disable translation in authselect as it requires multiple packages to be moved to core

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add nss-mdns requested by redis team

###### Change Log  

 - Add new package nss-mdns to Mariner
 - Bring authselect to core as it is direct dependency for nss-mdns
 - Disable translation in authselect as it requires multiple packages to be moved to core

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Local Build
-  Pipeline build id: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=509583&view=results

### CODE REVIEW

File: SPECS/nss-mdns/nss-mdns.spec [hunk 1]
```diff
@@ -0,0 +1,229 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+
+%undefine mariner_module_ldflags
+
+Name: nss-mdns
```
Comment (mfrw) [inline SPECS/nss-mdns/nss-mdns.spec:6]: This package was recently contributed by a redis engineer for 2.0
I think that we should ensure all the options are in parity from 2.0.

Reference: https://github.com/microsoft/CBL-Mariner/pull/7407

File: SPECS/nss-mdns/nss-mdns.signatures.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "nss-mdns-0.15.1.tar.gz": "ddf71453d7a7cdc5921fe53ef387b24fd0c3c49f4dcf94a2a437498596761a21"
+ }
+}
```
Comment (mfrw) [inline SPECS/nss-mdns/nss-mdns.signatures.json:5]:
```suggestion
}

```

File: SPECS/authselect/authselect.spec [hunk 3]
```diff
@@ -0,0 +1,371 @@
+# Do not terminate build if language files are empty.
+%define _empty_manifest_terminate_build 0
+
+Name:           authselect
```
Comment (mfrw) [inline SPECS/authselect/authselect.spec:4]: `> Disable translation in authselect as it requires multiple packages to be moved to core`

I think we should confirm this with @adobrzhansky - if this is a breaking change for their usecase.

Comment (adobrzhansky) [reply SPECS/authselect/authselect.spec:4]: I see no issues that may be related with that as long as nss package is working

**Review by mfrw — APPROVED** [review]

**Pull request merged**